src/components/Dataset/index.js

```diff
@@ -50,7 +50,7 @@ class Dataset extends React.Component {
 
   render() {
     const { match } = this.props;
-    const { data, loading, error } = this.state;
+    const { data, loading } = this.state;
```

Comment:
Have you considered showing the error instead? It seems like it would be a good service to the user to show it if something went wrong.

Reply:
To be honest, I just forgot to ask about that.

Reply:
Morten, I added a wrapper with notification method. Could you please check the implementation.